approaches/byte-pair-encoding.md

```diff
@@ -5,3 +5,5 @@ layout: coming_soon
 title: Byte-pair encoding
 description: A data compression algorithm for machine translation
 ---
+
+Tokenisation is fundamental to understanding human language. Algorithms for tokenisation can be based on word, subword or character. In this article, we discuss about subword-based tokenisation followed by Byte-pair encoding (BPE) algorithm.
```

Comment:
Thanks for starting this article! When writing the rest of it, please remember that you should aim for [minimal content](https://machinetranslate.org/style#minimal-content).